pulsar-broker/src/test/java/org/apache/pulsar/broker/admin/AdminApiHealthCheckTest.java

```diff
@@ -55,16 +61,91 @@ public void cleanup() throws Exception {
 
     @Test
     public void testHealthCheckup() throws Exception {
-        admin.brokers().healthcheck();
+        final int times = 30;
+        new Thread(() -> {
```

Comment:
Can we ensure that this Thread is done in the end of the trst? Otherwise we are leaking resources. 
I would use the Executor provided by the Pulsar service  to run this activity